### ISSUE TITLE
Simple data classes without executable code get flagged for EmptyBlock rule

### ISSUE THREAD

**Issue by LievenDeFoor** [Riverside-Software/sonar-openedge#1021]
```
USING ....IParameterSet FROM PROPATH.

BLOCK-LEVEL ON ERROR UNDO, THROW.

CLASS ....HCProviderQueryParameterSet
    IMPLEMENTS IParameterSet
    SERIALIZABLE:

    DEFINE PUBLIC PROPERTY ReferenceDate AS DATE NO-UNDO
    GET.
    SET.

END CLASS.
```

The above class gets flagged for EmptyBlock rule.
Currently we need to add some comments as workaround to suppress the warning...

**Comment by gquerret** [comment]
I thought that empty classes were already ignored in this rule. I'll have a look.

**Comment by gquerret** [comment]
The current exclusion is for static properties. As your property is not static, then the issue is raised.

**Comment by LievenDeFoor** [comment]
> The current exclusion is for static properties. As your property is not static, then the issue is raised.

Will the above case be excluded as well?

**Comment by gquerret** [comment]
> Will the above case be excluded as well?

Yes

**Comment by gquerret** [comment]
Fixed in develop
